R/summariseClinicalRecords.R

```diff
@@ -160,7 +160,9 @@ summariseClinicalRecord <- function(omopTableName,
   # Counts summary ----
   cli::cli_inform(c("i" = "Summarising {.pkg {omopTableName}} counts and records per person"))
   result <- summariseRecordsPerPerson(
-    omopTable, date, sex, ageGroup, recordsPerPerson)
+    omopTable, date, sex, ageGroup, recordsPerPerson)|>
+    omopgenerics::newSummarisedResult(settings = createSettings("")|>
```

Comment:
is this not automatically done when omopgenerics::newSummarisedResult?

Reply:
it is not happening, i think this is because "result" is the union of a list of  PatientProfiles::summariseResults and the first time this function is called, no strata is applied.

Reply:
try without the mutate